src/components/notifications/AppNotifications/AppNotificationItem.tsx

```diff
@@ -1,7 +1,13 @@
-import { useState } from 'react'
+import { useContext, useEffect, useState } from 'react'
 import { useFormattedTime } from '../../../utils/hooks'
 import CircleIcon from '../../general/Icon/CircleIcon'
 import './AppNotifications.scss'
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import { LazyMotion, PanInfo, domMax, m, useAnimationControls } from 'framer-motion'
+import ClearIcon from '../../../assets/ClearIcon.png'
+import UnreadIcon from '../../../assets/UnreadIcon.png'
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import { AppNotificationDragContext, AppNotificationsDragContext } from './index'
```

Comment:
I think it's expecting you to import the type as such:

import type { AppNotificationsDragContext } from './index'

Reply:
Correct, but doing so gives another eslint issue, I'll try taking a look and hashing it out